Lombiq.HelpfulLibraries.OrchardCore/Mvc/ShapeResultExtensions.cs

```diff
@@ -1,5 +1,5 @@
+using Lombiq.HelpfulLibraries.Common.Utilities;
```

Comment:
I'm torn between having to add this using statement to every file that wants to use `StringHelper.CreateInvariant()` and not having to.

What do you think about moving (at least) this method into the `System` namespace?